app/javascript/components/cloud-networks/update-nsxt-cloud-network-form.schema.js

```diff
@@ -0,0 +1,26 @@
+import { componentTypes, validatorTypes } from '@@ddf';
```

Comment:
As the schema generator is a function and there's an overlap between the create/update forms, I'd rather have a single schema and a parameter for the generator function to distinguish between the two versions. If there's a specific field in the initialValues that differs in create/update, then you can even get away without any logic in your code and use the `condition` property on some of the DDF entities.